types/externs.d.ts

```diff
@@ -175,6 +175,12 @@ declare global {
       throttlingMethod?: 'devtools'|'simulate'|'provided';
       /** The throttling config settings. */
       throttling?: ThrottlingSettings;
+      /** A user-facing string to display in Runtime Settings for the Device row */
+      providedDeviceString?: string | null;
+      /** A user-facing string to display in Runtime Settings for the Network throttling row */
+      providedNetworkThrottlingString?: string | null;
+      /** A user-facing string to display in Runtime Settings for the CPU throttling row */
+      providedCPUThrottlingString?: string | null;
```

Comment:
we've stuck to

```suggestion
      providedCpuThrottlingString?: string | null;
```
style for all the other settings with acronyms/initialisms